### PR TITLE
Add releasenotes and projecturl in the nuget package.

### LINKED ISSUE #25255: Change `releaseNotes` in Sgen package's nuspec
The current releaseNotes points to the release notes of .NET Core runtime 1.0.0 release. We should update it to a new fwlink.

### PR DESCRIPTION
**Pull request by huanwu**
Fix #25255 

@shmao @zhenlan @mconnew

### CODE REVIEW

File: src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj [hunk 1]
```diff
@@ -5,6 +5,8 @@
     <ProjectReference Include="..\src\Microsoft.XmlSerializer.Generator.csproj" />
   </ItemGroup>
   <PropertyGroup>
+    <ReleaseNotes>https://go.microsoft.com/fwlink/?LinkID=863421</ReleaseNotes>
```
Comment (zhenlan) [inline src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj]: Please forward this to https://github.com/dotnet/announcements/issues/39.

Comment (huanwu) [reply src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj]: Done

File: src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj [hunk 2]
```diff
@@ -5,6 +5,8 @@
...
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=858594</projectUrl>
```
Comment (shmao) [inline src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj]: Should the first `p` be capital?

Comment (huanwu) [reply src/Microsoft.XmlSerializer.Generator/pkg/Microsoft.XmlSerializer.Generator.pkgproj]: I will update to use capital P. But from https://docs.microsoft.com/en-us/nuget/schema/nuspec, it seems using lower p. Both should work.

**Pull request merged**